packages/relay/src/lib/relay.ts

```diff
@@ -19,47 +19,113 @@
  */
 
 import dotenv from 'dotenv';
-import findConfig from 'find-config';
-dotenv.config({ path: findConfig('.env') || '' });
-import { Relay, Eth, Net, Web3, Subs } from '../index';
-import { Web3Impl } from './web3';
+import { Logger } from 'pino';
 import { NetImpl } from './net';
 import { EthImpl } from './eth';
 import { Poller } from './poller';
-import { SubscriptionController } from './subscriptionController';
+import { Web3Impl } from './web3';
+import EventEmitter from 'events';
+import constants from './constants';
+import findConfig from 'find-config';
+import HbarLimit from './hbarlimiter';
 import { Client } from '@hashgraph/sdk';
-import { Logger } from 'pino';
+import { prepend0x } from '../formatters';
 import { MirrorNodeClient } from './clients';
 import { Gauge, Registry } from 'prom-client';
+dotenv.config({ path: findConfig('.env') || '' });
```

Comment:
Please revert this change and move it back to the top before importing any files

Reply:
updated